kratos/python/add_geometrical_utilities_to_python.cpp

```diff
@@ -0,0 +1,352 @@
+//
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Riccardo Rossi
+//
+
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/define_python.h"
+#include "processes/process.h"
+#include "python/add_geometrical_utilities_to_python.h"
+
+//Geometrical utilities
+#include "utilities/normal_calculation_utils.h"
+#include "utilities/body_normal_calculation_utils.h"
+#include "utilities/body_distance_calculation_utils.h"
+#include "utilities/signed_distance_calculation_utils.h"
+// #include "utilities/signed_distance_calculator_bin_based.h"
+#include "utilities/parallel_levelset_distance_calculator.h"
+#include "utilities/brute_force_point_locator.h"
+#include "utilities/deflation_utils.h"
+#include "utilities/binbased_fast_point_locator.h"
+#include "utilities/binbased_fast_point_locator_conditions.h"
+#include "utilities/binbased_nodes_in_element_locator.h"
+#include "utilities/embedded_skin_utility.h"
+#include "utilities/geometry_tester.h"
+#include "utilities/cutting_utility.h"
+#include "utilities/geometrical_transformation_utilities.h"
+#include "utilities/iso_printer.h"
+#include "utilities/interval_utility.h"
+#include "utilities/convect_particles_utilities.h"
```

Comment:
Maybe makes sense to move this one to the others too (not sure at all)

Reply:
The utility moves the mesh from time t_n to t_(n+1); to me it fits in geometrical utilities...